kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java

```diff
@@ -94,19 +91,17 @@ public void updateConnector() {
     @CaseId(195)
     @Test
     public void deleteConnector() {
-        pages.openConnectorsList(CLUSTER_NAME)
-                .waitUntilScreenReady()
-                .openConnector(CONNECTOR_FOR_DELETE.getName());
-        pages.connectorsView.clickDeleteButton();
-        pages.openConnectorsList(CLUSTER_NAME);
-        Assertions.assertFalse(pages.connectorsList.isConnectorVisible(CONNECTOR_FOR_DELETE.getName()),"isConnectorVisible()");
+        connectorsView.goTo(CLUSTER_NAME, CONNECTOR_FOR_DELETE.getName())
```

Comment:
we don't need to remove navigation steps from test. please return mainPage.goTo()
.goToSideMenu(CLUSTER_NAME, bla-bla

Reply:
Done